spec/requests/get_players_spec.rb

```diff
@@ -7,12 +7,12 @@
   end
 
   context 'when there are players in the database' do
-    it 'returns the list of players' do
-      francis = create(:player, name: 'francis')
+    let!(:player) { create(:player, name: 'francis') }
 
+    it 'returns the list of players' do
       get '/players.json'
       body = JSON.parse(response.body, symbolize_names: true)
-      expect(body[:players][0]).to include(name: francis.name)
+      expect(body[:players]).to include(hash_including(name: player.name))
```

Comment:
Use hardcoded expected values instead of being dynamic to avoid false positives (aka here if `player.name` is `nil` for a weird reason you want that test to fail).